### PR TITLE
Consolidate Icons to Steeze-UI

### PR DESCRIPTION
**Pull request by ItsMeBrianD**
### Description

This replaces all inline-svg icons and svelte-icons usage with `steeze-ui` and `tabler-icons`. 
This gives contributors a clear path to using icons, with a pre-selected pack to keep a consistent look and feel.

- Steeze-UI: https://github.com/steeze-ui/icons/tree/main
- Tabler Icons: https://tabler-icons.io/

This addresses #533 

### Checklist

- [ ] For UI or styling changes, I have added a screenshot or gif showing before & after
- [x] I have added a [changeset](https://github.com/evidence-dev/evidence/blob/main/CONTRIBUTING.md#adding-a-changeset)

### CODE REVIEW

File: packages/core-components/src/lib/unsorted/ui/BreadCrumbs.svelte [hunk 1]
```diff
@@ -62,22 +63,26 @@
...
 		{:else}
 			{#each crumbs as crumb, i}
 				{#if i > 0}
 					&emsp13;/&emsp13;<a href={crumb.href}>{crumb.title}</a>
 				{:else}
-					<a href={crumb.href}>
+					<a href={crumb.href} class="inline-flex gap-1 items-center">
 						{#if crumb.title === 'Home'}
-							<HomeIcon height="14" width="14" verticalOffset="3" />
+							<Icon src={Home} class="h-4 w-4 inline-block" />
```
Comment (csjh) [inline packages/core-components/src/lib/unsorted/ui/BreadCrumbs.svelte:85]: Is it possible to change the `stroke-width`? 2 is a bit out of place, somewhere in 1-1.5 fits a bit nicer imo

File: packages/core-components/src/lib/unsorted/viz/Value.svelte [hunk 2]
```diff
@@ -86,79 +88,19 @@
...
-		<span class="error-label">Error</span>
-		<span class="additional-info-icon">
-			<HelpCircleIcon height="18" width="18" verticalOffset="1" color="--grey-100" />
-		</span>
-		<span class="error-msg">{error}</span>
+	<span
+		class="group inline-flex gap-1 items-center relative cursor-help text-white font-sans text-sm bg-red-700 rounded-2xl pl-2 pr-[1px] mx-0.5"
+	>
+		<span class="inline pl-1">Error</span>
+		<Icon src={HelpCircle} class="w-6 h-6 text-gray-100 pb-0.5 pt-[1px]" />
```
Comment (csjh) [inline packages/core-components/src/lib/unsorted/viz/Value.svelte:95]: Seems like the error view is quite a bit bigger now, is that intentional?

**Comment by archiewood** [comment]
Screenshots would be extra helpful for reviewing this PR :)

**Comment by ItsMeBrianD** [comment]
> Screenshots would be extra helpful for reviewing this PR :)

Planning on grabbing those first thing this morning, ran out of time yesterday

File: packages/core-components/src/lib/unsorted/viz/Value.svelte (hunk 2 above)
Comment (ItsMeBrianD) [reply packages/core-components/src/lib/unsorted/viz/Value.svelte:95]: I built them side-by-side so I'm not sure where that would be coming from; is it the icon or the entire error view that you're seeing issues with?

**Comment by ItsMeBrianD** [comment]
image

image

image

image

image

image

image

image

image

image

image

image

image

image

**Comment by archiewood** [comment]
Like the icon pack. 
Looking forward to not having to find new custom ones for every feature.

A couple of nits, mainly on positioning

- **DB form help icons** - can we align vertically with text - icon seem a bit low
- **DB form info icons** - can we use one with an `i` rather than a `?`
- **Custom format grid check/cross** - can we align vertically with text - icon seem a bit low
- **External Link Icons** - should we match the text color, seems like the icon is a bit darker
- **DataTable Sort Icons** - can we align vertically with text - icon seem a bit high

**Comment by ItsMeBrianD** [comment]
image
image
image
image

**Comment by archiewood** [comment]
Epic, Epic, Epic, Epic

**Pull request merged**